Sources/System/Internals/CInterop.swift

```diff
@@ -9,9 +9,12 @@
 
 #if os(macOS) || os(iOS) || os(watchOS) || os(tvOS)
 import Darwin
-#elseif os(Linux) || os(FreeBSD) || os(OpenBSD) || os(Android)
+#elseif canImport(Glibc)
```

Comment:
This is a big switch from OS name to capability. Would it make sense to move this to _after_ the `#elseif os(Windows)` branch?

Reply:
We should do the same in the other files where there's a sequence of if-os and if-canImport.